community/modules/file-system/DDN-EXAScaler/variables.tf

```diff
@@ -203,20 +203,52 @@ variable "boot" {
 # family: image family name
 # name: !!DEPRECATED!! - image name
 variable "image" {
-  description = "Source image properties"
+  description = "DEPRECATED: Source image properties"
   type        = any
   # Ommiting type checking so validation can provide more useful error message
   # type = object({
   #   project = string
   #   family  = string
   # })
+  default = {
```

Comment:
nit. Can we simply `default = null` here ?

Reply:
I agree, at above nit level. Please change to a simple `null`.

Reply:
Done